Clarify feature support (#21)
mulesoft/docs-govcloud: modules/ROOT/pages/gov-cloud-standalone.adoc

> = Standalone Mule Support

Reviewer: should the title specifically call out that this is for GovCloud?

"Standalone Mule Support in MuleSoft Government Cloud"

Author: Updated - thanks!

ifndef::env-site,env-github[]
include::_attributes.adoc[]
endif::[]

> MuleSoft Government Cloud deployment permits deployment of Mule runtime engines into an on-premises environment (standalone Mules). Standalone Mules, and

Reviewer 2: ```suggestion
MuleSoft Government Cloud deployment permits deployment of Mule runtime engine instances into an on-premises environment (standalone Mule instances). Standalone Mule instances, and
```

> MuleSoft support for standalone Mules, are outside the scope of FedRAMP. The customer is responsible for ensuring that all data shared with Salesforce complies

Reviewer 2: ```suggestion
MuleSoft support for them, are outside the scope of FedRAMP. The customer is responsible for ensuring that all data shared with Salesforce complies
```

with any and all data and information protection policies, requirements, and standards. Review the MuleSoft System Security Plan for FedRAMP scope.

> Rigorous internal security and compliance reviews were performed on services that support standalone Mule deployments in MuleSoft Government Cloud. 

Reviewer 2: "were performed on"? Hmm...is this any better:
```suggestion
Services that support standalone Mule deployments in MuleSoft Government Cloud are rigorously reviewed to help ensure internal security and compliance.
```
I suggest "help ensure" because "ensure" alone opens us up to legal obligations. Sheese.

> The Significant Change Request Form and updated System Security Plan were approved by our FedRAMP sponsoring agency.

Reviewer 2: I would switch the order here as well ^

> For FIPS compliance, Mule Runtime version 4.1.x or later and runtime agent version 2.4.9 or later are required.

Reviewer 2: These are per our Product Naming guidelines:
```suggestion
For FIPS compliance, Mule v4.1.x or later and Anypoint Runtime Manager agent v2.4.9 or later are required.
```
v or version, either is okay; we haven't decided that one yet....

> Standalone Mule support enables federal, state, and local government customers to deploy Mules inside their private cloud or 

Reviewer 2: ```suggestion
Support for standalone Mule instances enables federal, state, and local government customers to deploy Mule instances inside their private cloud or 
```

on-premises environment and manage them using the MuleSoft Government Cloud control plane. Key features include:

> * Mule Management

Reviewer 2: ```suggestion
* Mule management
```

** xref:runtime-manager::index.adoc[Anypoint Runtime Manager] capabilities

* Mule Monitoring

** Dashboards for apps, servers, and server groups
** Visibility into usage and performance metrics
** Email alerts and notifications
** Performance metrics for flows
** xref:runtime-manager::monitoring.adoc[Monitor Applications and Servers] 

== Mule Version Support

The following Mule versions are supported for standalone Mule support in MuleSoft Government Cloud:

> * 4.0.x

Reviewer: @gh0stl1m can we get confirmation on these specific mule versions that are supported in GovCloud?

Reviewer: @paigehargrave we can remove 4.0.x from this list for now

> * Anypoint Runtime Manager Monitoring

Reviewer 2: ```suggestion
* Anypoint Runtime Manager monitoring
```

> ** xref:runtime-manager::monitoring.adoc[Monitor Applications and Servers] 

Reviewer 2: ```suggestion
** xref:runtime-manager::monitoring.adoc[Ability to monitor applications and servers] 
```

> The following Mule versions are supported for standalone Mule support in MuleSoft Government Cloud:

Reviewer 2: ```suggestion
The following Mule versions are supported as standalone instances in MuleSoft Government Cloud:
```

= Standalone Mule Support in MuleSoft Government Cloud
ifndef::env-site,env-github[]
include::_attributes.adoc[]
endif::[]

MuleSoft Government Cloud deployment permits deployment of Mule runtime engines into an on-premises environment (standalone Mules). Standalone Mules, and
MuleSoft support for standalone Mules, are outside the scope of FedRAMP. The customer is responsible for ensuring that all data shared with Salesforce complies
with any and all data and information protection policies, requirements, and standards. Review the MuleSoft System Security Plan for FedRAMP scope.

Rigorous internal security and compliance reviews were performed on services that support standalone Mule deployments in MuleSoft Government Cloud. 
The Significant Change Request Form and updated System Security Plan were approved by our FedRAMP sponsoring agency.

> For FIPS compliance, Mule 4 agent version 2.4.9 or later is required.

Reviewer: "For FIPS compliance, Mule Runtime version 4.1.x or later, along with runtime agent version 2.4.9 or later is required."

Author: Updated this.

= Standalone Mule Support in MuleSoft Government Cloud
ifndef::env-site,env-github[]
include::_attributes.adoc[]
endif::[]

MuleSoft Government Cloud deployment permits deployment of Mule runtime engines into an on-premises environment (standalone Mules). Standalone Mules, and

> MuleSoft support for standalone Mules, are outside the scope of FedRAMP. You are solely responsible for ensuring that all data shared with Salesforce complies

Reviewer: Can we clarify that it is the Customer's responsibility? I think that's important from the legal side

Author: Updated this.

with any and all data and information protection policies, requirements, and standards. Review the MuleSoft System Security Plan for FedRAMP scope.

Rigorous internal security and compliance reviews were performed on services that support standalone Mule deployments in MuleSoft Government Cloud. 
The Significant Change Request Form and updated System Security Plan were approved by our FedRAMP sponsoring agency.

For FIPS compliance, Mule 4 agent version 2.4.9 or later is required.

Standalone Mule support enables federal, state, and local government customers to deploy Mules inside their private cloud or 
on-premises environment and manage them using the MuleSoft Government Cloud control plane. Key features include:

* Mule Management

** Unified view of applications, servers, and APIs
** Application deployment and management across all environments
** Integration with continuous deployment pipelines
** xref:runtime-manager::index.adoc[Anypoint Runtime Manager] capabilities

> * Mule Monitoring

Reviewer: Can we change the title of this section to: "Anypoint Runtime Manager Monitoring"

** Dashboards for apps, servers, and server groups
** Visibility into usage and performance metrics
** Email alerts and notifications
** Performance metrics for flows
** xref:runtime-manager::monitoring.adoc[Monitor Applications and Servers] 

== Mule Version Support

The following Mule versions are supported for standalone Mule support in MuleSoft Government Cloud:

* 4.0.x
* 4.1.x
* 4.2.x
* 4.3.x

> == Standalone Mules and FedRAMP boundary

Reviewer: We can remove this section since it's covered in the first paragraph

Author: Removed.
